Cargo.toml

```diff
@@ -103,6 +103,7 @@ futures-util = "0.3.25"
 googletest = "0.10"
 hostname = { version = "0.4.0" }
 http = "0.2.9"
+http-serde = { version = "1.1.2" }
```

Comment:
A pity that we can't use `2.2.1` which already contains serializers for `Option<Version>`, for example.